### ISSUE TITLE
Add XGBoost

### ISSUE THREAD

**Issue by gran4** [gran4/AI-InvestiBot#74]
**Is your feature request related to a problem? Please describe.**
The ai is not as accurate as one would hope.

**Describe the solution you'd like**
Add XGBoost and mix it with tensorflow. This would increase accuracy by build multiple models one after the other. Each new model tries to correct the mistakes made by the previous models.
